README.md

```diff
@@ -401,6 +401,9 @@ Reconciler A reacts to this annotation by creating a `ServiceBindingProjection`
 
 Reconciler B **MUST** set the `ServiceBindingProjection`'s `Ready` condition according to the rules set in [Ready Condition Status](#ready-condition-status). Reconciler A **MUST** reflect `ServiceBindingProjection`'s `Ready` condition with an additional `ProjectionReady` condition on `ServiceBinding`.  The `Ready` condition of `ServiceBinding` **MUST NOT** be `True` unless `ProjectionReady` is `True`.
 
+An implementation of the Custom Projection extension **MAY** allow defining a `.spec.application` referencing a non-`PodSpec`-able resource in addition to the mechanisms to define an application reference described in the [Service Binding][sb] section.
```

Comment:
This change is allowed in the `ServiceBinding` resource and not the `ServiceBindingProjection` resource, right?
Please clarify that with an additional sentence.

Reply:
The primary desire for the `ServiceBindingProjection` is to have a way for non-PodSpecable resources to be the target of the ServiceBinding's application reference. Strictly speaking, the PodSpecable requirement doesn't exist on the ServiceBindingProjection resource, so there is nothing to relax, but I agree that a clarification is worthwhile.

Reply:
This requirement applies to the ServiceBinding resource. Saying "An implementation of the Custom Projection extension" is redundant because it's in the Custom Projection section, it could only ever apply to the Custom Projection implementation.

```suggestion
A `ServiceBinding` with a custom projection **MUST** allow defining a `.spec.application` referencing a non-`PodSpec`-able resource in addition to the mechanisms to define an application reference described in the [Service Binding][sb] section.
```

Reply:
This change avoids the confusion!

Reply:
Thanks @baijum and @scothis. I committed the suggestions